packages/auth-types/index.d.ts

```diff
@@ -238,6 +238,10 @@ export interface OAuthCredential extends AuthCredential {
   secret?: string;
 }
 
+export class AuthSettings {
```

Comment:
Let's make this an interface since this can't be instantiated externally anyway.
`export interface AuthSettings{`